mscore/editstyle.cpp

```diff
@@ -557,6 +558,11 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
             mapper2->setMapping(sw.widget, int(sw.idx));
             }
 
+      int topBottomMargin = automaticCapitalization->rect().height() - preferences.getInt(PREF_UI_THEME_FONTSIZE);
+      topBottomMargin /= 2;
+      topBottomMargin = topBottomMargin > 4 ? topBottomMargin - 4 : 0;
+      automaticCapitalization->layout()->setContentsMargins(9, topBottomMargin, 9, topBottomMargin);
+
```

Comment:
Does this fix really need to be done in code?

I'm asking because doing it this way makes it difficult to maintain. Someone in the future might make changes to the UI that conflict with these in-code alterations and then not be able to figure out why things aren't working as they “should”.

Reply:
The easiest way to go about would be to set the margins to 0 in the .ui file (but that just disables margins and I don't know if that is acceptable or not). It could be set it to a fixed lower value, but as @Jojo-Schmitz pointed out, his MuseScore instance had a different amount of clipping, which means that a fixed value (other than 0) is not really great.